Realm/Realm/Handles/AppHandle.cs

```diff
@@ -232,7 +232,7 @@ public string GetRealmPath(User user, string partition)
             });
         }
 
-        public SessionHandle GetSessionForPath(SharedRealmHandle realm)
+        public SessionHandle GetSessionForRealm(SharedRealmHandle realm)
```

Comment:
I noticed this while looking for other cases where we might not be disposing properly of the sync session. The name was a leftover from when it had a string argument and figured it no longer reflected the current signature.